functions/Stop-DbaXESession.ps1

```diff
@@ -89,7 +89,7 @@ function Stop-DbaXESession {
 					try {
 						$xe.Stop()
 					} catch {
-						Write-Error -Message "Could not stop XEvent Session $session on $instance"
+						Stop-Function -Message "Could not stop XEvent Session on $instance" -Target $session -ErrorRecord $_
```

Comment:
One more minor adjustment I didn't catch until I looked at the whole command. We need to add `-Continue` switch to `Stop-Function` call so it won't stop the foreach loop and keep going to the next one in the collection. If you can do that on each one I'll get this merged.

Reply:
ah, yup, makes sense.  Done in the latest commit